docs/tutorials/08-tutorial-sug-roc-notleap.ipynb

```diff
@@ -0,0 +1,981 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "21fc58f4-c6fe-4555-9022-f8d3739cb8dd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-10-09 02:10:16 PM Note: detected 128 virtual cores but NumExpr set to maximum of 64, check \"NUMEXPR_MAX_THREADS\" environment variable.\n",
+      "2023-10-09 02:10:16 PM Note: NumExpr detected 128 cores but \"NUMEXPR_MAX_THREADS\" not set, so enforcing safe limit of 8.\n",
+      "2023-10-09 02:10:16 PM NumExpr defaulting to 8 threads.\n",
+      "2023-10-09 02:10:16 PM Warning on use of the timeseries module: If the inherent timescales of the system are long compared to those being analyzed, this statistical inefficiency may be an underestimate.  The estimate presumes the use of many statistically independent samples.  Tests should be performed to assess whether this condition is satisfied.   Be cautious in the interpretation of the data.\n",
+      "2023-10-09 02:10:16 PM JAX detected. Using JAX acceleration.\n",
+      "2023-10-09 02:10:17 PM Moving :ROC@C14x (1 atoms) to the origin...\n",
+      "2023-10-09 02:10:17 PM Aligning :ROC@C8x (1 atoms) with the z axis...\n"
+     ]
+    }
+   ],
+   "source": [
+    "from paprika.build import align\n",
+    "\n",
+    "structure = pmd.openmm.load_topology(\n",
+    "    model.topology,\n",
+    "    system,\n",
+    "    xyz=model.positions,\n",
+    "    condense_atom_types=False,\n",
+    ")\n",
+    "\n",
+    "# Specifically, put the line between atoms C14x and C8x on guest residue \"ROC\" in line with the z axis. Can be changed to other guest residues.\n",
+    "G1 = \":ROC@C14x\"\n",
+    "G2 = \":ROC@C8x\"\n",
+    "\n",
+    "aligned_structure = align.translate_to_origin(structure)\n",
+    "aligned_structure = align.zalign(aligned_structure, G1, G2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65c3b296-18a7-43dd-8977-61eda6f0ad8a",
+   "metadata": {},
+   "source": [
+    "#### Add solvent and dummy atoms"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "675d0486-5edc-48b2-ae9a-e4672b06e134",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = Modeller(\n",
+    "    topology.to_openmm(ensure_unique_atom_names=True), aligned_structure.positions\n",
+    ")\n",
+    "model.topology.setUnitCellDimensions(unit_cell_dimensions)\n",
+    "model.addSolvent(\n",
+    "    ff, boxSize=unit_cell_dimensions, model=\"tip3p\", ionicStrength=5 * millimolar\n",
+    ")  # Here salt can be added or removed\n",
+    "system = ff.createSystem(\n",
+    "    model.topology,\n",
+    "    nonbondedMethod=PME,\n",
+    "    nonbondedCutoff=0.9 * nanometers,\n",
+    "    rigidWater=True,\n",
+    "    constraints=HBonds,\n",
+    ")  # We recreate the OpenMM system because it now has waters\n",
+    "\n",
+    "# Set the nonbonded forces for the dummy atoms to have force constants of 0 and masses of 0.\n",
```

Comment:
Hmm, if you set the masses to zero, then the dummy particles won't move at all throughout the simulation. Though we do apply strong harmonic positional restraints on them, I'm not sure what the effect of this is. I usually set the masses to the mass of lead -- 207 Dalton.

Also, with `NonbondedForce` we are setting the partial charge and LJ parameters to zero not the force constant.

Reply:
Good point; I changed the mass from 207.2 to 0 at some point to see if there was any effect and never changed it back. I'll also update the comment to reflect what we're actually changing.